packages/get_line/get_line.4.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "get_line"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+homepage: "https://github.com/UnixJunkie/get_line"
+bug-reports: "https://github.com/UnixJunkie/get_line/issues"
+dev-repo: "git://github.com/UnixJunkie/get_line"
+license: "GPL"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml"
+  "jbuilder" {build}
```

Comment:
I think you can use `dune` here. It installs the retro-comptible `jbuilder` binary